db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/OperatorToString.java

```diff
@@ -968,7 +968,7 @@ else if (operand instanceof ConditionalNode) {
                             methodName.equals("ADD_MONTHS")    ||
                             methodName.equals("ADD_YEARS")     ||
                             methodName.equals("ADD_DAYS")      ||
-                            methodName.equals("ROUND"))
+                            methodName.equalsIgnoreCase("ROUND"))
```

Comment:
Shouldn't that apply to every other item in this list?

Reply:
Won't need to do that. Other functions were "made in splice" and their method names were uppercase. The round function came from Splice package (uppercase) and java.lang.StrictMath (lowercase), we need to handle this one especially.

Reply:
All right! Could you add a comment in the code to explain that?

Reply:
Done